recipes/django/meta.yaml

```diff
@@ -0,0 +1,234 @@
+{% set version = "1.8.11" %}
+
+package:
+    name: django
+    version: {{ version }}
+
+source:
+    fn: Django-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/D/Django/Django-{{ version }}.tar.gz
+    md5: 1c85d0d582dc211adc6851dd2dc86228
+    patches:
+        # Hard-code gdal and geos' paths to the corresponding conda packages.
+        - libgdal.patch
+        - libgeos.patch
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    entry_points:
+        - django-admin = django.core.management:execute_from_command_line
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - gdal >=1.11.2,<2.0  # [not win and not py35]
```

Comment:
Actually no. Just tested it locally and it has the same effect. I thought that the line should be `True` if `unix` and `py<35` are True, but It is evaluating to `True` when only one of them is (`py<35`) and hence unix and python 3.5 gets dropped too..